### ISSUE TITLE
[table-v2] [Accessibility WCAG section508] - el-table-v2 

### ISSUE THREAD

**Issue by hugo-valcourt** [element-plus/element-plus#11732]
Bug Type: **`Other`**

## Environment

- Vue Version: `3.2.47`
- Element Plus Version: `2.2.32`
- Browser / OS: `110.0.5481.177 (arm64) / MacOs`
- Build Tool: `Vite`

### Related Component

- `el-table-v2`

### Reproduction Link

[Docs](https://element-plus.org/en-US/component/table-v2.html)

### Steps to reproduce

With a WCAG validator.

- In chrome, dev tools, lighthouse, you can check the Accessibility
- In storybook with the Accessibility Tab

### What is Expected?

Should not give accessibility error or warning

### What is actually happening?

Certain ARIA roles must contain particular children

Ensures elements with an ARIA role that require child roles contain them

https://dequeuniversity.com/rules/axe/4.6/aria-required-children?application=axeAPI
.el-table-v2__table

Critical
Required ARIA children role not present: rowgroup, row
Serious

*****

div.el-table-v2__table.el-table-v2__main
Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children.
Some ARIA parent roles must contain specific child roles to perform their intended accessibility functions. 
https://dequeuniversity.com/rules/axe/4.4/aria-required-children?utm_source=lighthouse&utm_medium=devtools
Element has no aria-busy="true" attribute
cat.aria wcag2a wcag131

### Additional comments

Should always pass wcag and section508 accessibility. You could provide accessibility default in the component. And maybe add some properties to override them. In some juridictions, the law force us to follow accessibility rules, so it's very important.

Thanks for the good work
